### PR TITLE
Fluxtomize with multiple levels of subdirectories

### PR DESCRIPTION
**Pull request by ansarhun**
Allow fluxtomize to process nested subdirectories

Relates to: https://github.com/allenporter/flux-local/issues/295

### CODE REVIEW

**Comment by allenporter** [comment]
Thank you didn't realize you also sent a PR in addition to an issue -- I will take a look today.

**Comment by allenporter** [comment]
Oh i see this just came in today, great

File: flux_local/kustomize.py [hunk 1]
```diff
@@ -251,7 +251,7 @@ async def fluxtomize(path: Path) -> bytes:
     filenames.sort()
     for filename in filenames:
         new_path = path / filename
-        if new_path.is_dir() and await can_kustomize_dir(new_path):
+        if new_path.is_dir():
```
Comment (allenporter) [inline flux_local/kustomize.py:254]: This is the only purpose of `can_kustomize_dir` so can you remove it and the test?

**Comment by allenporter** [comment]
Thank you for the PR, I adjusted the description a bit and it looks good overall. Do you mind removing the dead code also then we can merge and i'll release a new version of the library with your fix.

File: flux_local/kustomize.py (hunk 1 above)
Comment (ansarhun) [reply flux_local/kustomize.py:254]: This is still used in `Build.run` and as far as I understand this is still needed to decide if the kustomize or fluxtomize should be used

https://github.com/allenporter/flux-local/blob/main/flux_local/kustomize.py#L209
```python
        if not await can_kustomize_dir(self._path):
            # Attempt to effectively generate a kustomization.yaml on the fly
            # mirroring the behavior of flux
            return await fluxtomize(self._path)
```

Comment (allenporter) [reply flux_local/kustomize.py:254]: Ah! thank you I missed that.

**Review by allenporter — APPROVED** [review]

**Comment by ansarhun** [comment]
@allenporter thank you for the approval, I fixed the linter issue, the other issues seems to be related that the `cluster7` is not available on the `live` branch for the diff task, should I remove it from the GitHub workflow?

**Comment by allenporter** [comment]
> @allenporter thank you for the approval, I fixed the linter issue, the other issues seems to be related that the `cluster7` is not available on the `live` branch for the diff task, should I remove it from the GitHub workflow?

No, should be working as intended until next time. Thanks!

**Pull request merged**